src/main/java/cz/muni/fi/pa165/entity/User.java

```diff
@@ -1,22 +1,133 @@
 package cz.muni.fi.pa165.entity;
 
 /**
- * Hello world!
+ * Robo
  *
  */
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
 @Entity
-@Table(name="User")
-public class User
-{
+@Table(name="Users")
+public class User {
+
+
     @Id
-    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(nullable=false,unique=true)
-    @Pattern(regexp=".+@.+\\....?")
+    @ManyToOne
+    @NotNull
+    private Role role;
+
+    @OneToMany
+    @NotNull
+    private List<Booking> user = new ArrayList<Booking>();
```

Comment:
Podle diagramu ma booking instanci usera, user nepotrebuje mit seznam bookingu, protoze to pres DAO a JPQL muzeme zjistit, kdyz kazdy booking ma instanci usera